src/tools/generate/shared.js

```diff
@@ -33,6 +33,21 @@ const find_config_files = async () => {
   return await recursive('config', [filter]);
 };
 
+const get_config_files = async () => {
```

Comment:
I think we should change the function name to something like `get_yearly_configs` because function names `find_*_files` represent files (handle or name), not the content of the files. This one however, returns parsed object loaded from those files. Also, using the term `yearly` in the name will suggest that the object keys are years.

Reply:
Good idea! Done.